pkg/commands/git.go

```diff
@@ -1061,3 +1070,48 @@ func (c *GitCommand) BeginInteractiveRebaseForCommit(commits []*Commit, commitIn
 func (c *GitCommand) SetUpstreamBranch(upstream string) error {
 	return c.OSCommand.RunCommand(fmt.Sprintf("git branch -u %s", upstream))
 }
+
+func (c *GitCommand) AddRemote(name string, url string) error {
+	return c.OSCommand.RunCommand(fmt.Sprintf("git remote add %s %s", name, url))
```

Comment:
Maybe it's a good idea to refactor `c.OSCommand.RunCommand(fmt.Sprintf(...))` in it's own function

Reply:
I've made it so that RunCommand takes the same input as fmt.Sprintf